docs/cli-commands.md

```diff
@@ -265,8 +265,19 @@ by IRIs provided in a mapping file.
 dsp-tools id2iri xmlfile.xml mapping.json
 ```
 
+The following options are available:
+
+- `-r` | `--remove-resources` (optional): remove resources if their ID is in the mapping 
+  (this prevents doubled resources on the DSP server,
+  because the resources occurring in the mapping already exist on the DSP server)
+
```

Comment:
Maybe add here that there can't be further statements with the resource in subject position. Because to me that would not be obvious.

Reply:
I think we speak of 2 different things: You talk about triples (subject - predicate - object) in the database, and I talk about resources that can be uploaded (=`<resource>` tags in the XML file --> a resource like "Iliad Prooem" that can be linked at with a link like https://ark.dasch.swiss/ark:/72163/1/082E/40kW9f9=SzOnQiyvhBNSqw=.20220414T072754555597Z).

In the context of an xmlupload, we don't care about what kind of triples can be in the database. We only care about `<resource>`s that are uploaded. And if "Iliad Prooem" has been uploaded already, I don't want to upload it a second time.